.github/workflows/build-image.yml

```diff
@@ -292,9 +293,51 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  updates-img:
+    needs: pr-info
+    if: needs.pr-info.outputs.allowed_user == 'true' && contains(needs.pr-info.outputs.args, '--updates.img')
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+       STATUS_NAME: updates-img
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.pr-info.outputs.sha }}
+          fetch-depth: 0
+
+      - name: Install required packages
+        run: |
+          sudo apt install -y dracut
+
+      - name: Create updates image
+        run: |
+          ./scripts/makeupdates
```

Comment:
Hopefully this will work. Without the tag specification it takes the `latest` which might be problematic in some cases. Also outdated PR might broke this even more.

I wonder if building RPMs and just adding them to the updates image wouldn't be better solution.

In any case, great improvement - just a bit fragile.